packages/ui/src/views/chatmessage/ChatMessage.js

```diff
@@ -313,8 +595,62 @@ export const ChatMessage = ({ open, chatflowid, isDialog }) => {
 
     return (
         <>
-            <div className={isDialog ? 'cloud-dialog' : 'cloud'}>
-                <div ref={ps} className='messagelist'>
+            {isDragOver && getAllowChatFlowUploads.data?.isUploadAllowed && (
+                <Box className='drop-overlay'>
+                    <Typography variant='h2'>Drop here to upload</Typography>
+                    {getAllowChatFlowUploads.data.uploadFileSizeAndTypes.map((allowed) => {
+                        return (
+                            <>
+                                <Typography variant='subtitle1'>{allowed.fileTypes?.join(', ')}</Typography>
+                                <Typography variant='subtitle1'>Max Allowed Size: {allowed.maxUploadSize} MB</Typography>
+                            </>
+                        )
+                    })}
+                </Box>
+            )}
+            {isRecording && (
+                <Box className='drop-overlay'>
+                    <div className={'audio-recording-container'}>
+                        <Typography variant='h2'>Recording</Typography>
+                        <div className='recording-control-buttons-container'>
+                            <i className='cancel-recording-button'>
+                                <Button variant='outlined' color='error' onClick={onRecordingCancelled}>
+                                    Cancel
+                                </Button>
+                            </i>
+                            <div className='recording-elapsed-time'>
+                                <i className='red-recording-dot'>
+                                    <IconCircleDot />
+                                </i>
+                                <p id='elapsed-time'>00:00</p>
+                            </div>
+                            <i className='stop-recording-button'>
+                                <Button variant='outlined' color='primary' onClick={onRecordingStopped}>
+                                    Save
+                                </Button>
+                            </i>
+                        </div>
+                    </div>
+                    {recordingNotSupported && (
+                        <div className='overlay hide'>
+                            <div className='browser-not-supporting-audio-recording-box'>
+                                <p>To record audio, use browsers like Chrome and Firefox that support audio recording.</p>
+                                <button type='button' onClick={() => onRecordingCancelled()}>
+                                    Ok.
+                                </button>
+                            </div>
+                        </div>
+                    )}
+                </Box>
+            )}
+            <div
+                onDragOver={handleDragOver}
```

Comment:
Moved `onDragOver={handleDragOver}` to this `div`. The initial implementation of this doesn't work when expand chatbox, it caused the messages to not render properly: 
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/ceb134fa-2d19-413f-92a9-d5943d698ed4)

However, there are also issues moving to here. Unless we can find a way to solve this without spending too much time, otherwise we can remove this drag drop functionality